enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -775,7 +775,25 @@ To answer this question, we need to do the following:
 * Verify that OpenShift Service Mesh will include support for this feature in time for this EP.
 * Evaluate any potential security concerns around this feature.
 
-**Resolution**: TBD.
+**Resolution**: Yes. However, whether ReferenceGrants will be functional, is dependent on the
+OSSM version use in dev preview:
+
+- OSSM 2.3 uses Istio 1.14 and Istio 1.14 doesn't support ReferenceGrants; therefore,
```

Comment:
If Istio 1.14 supports v1beta1, why doesn't it support ReferenceGrants?  It makes me question what other parts of the v1beta1 API might be missing from Istio 1.14.

Reply:
That's a great question, it took me a ton of searching, but I think finally got to the bottom of it.

It just was never supported in Istio 1.14. See https://github.com/istio/istio/issues/39919 and https://github.com/istio/istio/pull/40007 which were implemented in Istio 1.15.

> It makes me question what other parts of the v1beta1 API might be missing from Istio 1.14.

It's reasonable to ask that question. I tested the core http, edge, passthrough functionality with OSSM 2.3 / Istio 1.14, so I feel confident that works, but the fact it seems like they shoehorned Gateway API v1beta1 back to Istio 1.14, makes me think there might be more unusual behavior with v1beta1.

Regardless, OSSM 2.3 really seems like a last resort, especially since Istio 1.14 is EOL. Let's hope OSSM 2.4 is ready soon.